aten/src/ATen/native/cuda/fused_adam_utils.cuh

```diff
@@ -112,56 +112,59 @@ struct FusedAdamMathFunctor {
       int chunk_size,
       FusedOptimizerTensorListMetadata<depth>& tl,
       const float* lr_ptr,
-      const double lr,
-      const double beta1,
-      const double beta2,
-      const double weight_decay,
-      const double eps,
-      const bool maximize,
-      const bool amsgrad,
+      const double& lr,
```

Comment:
Woah, how does making these refs minimize register usage?

Reply:
I'm not sure, I think this might not be the one but even so I think it's reasonable to make these arguments const ref